usl_pipeline/cloud_functions/main.py

```diff
@@ -21,7 +21,7 @@
 from usl_lib.storage import cloud_storage
 from usl_lib.storage import file_names
 from usl_lib.storage import metastore
-from usl_lib.shared import wps_variables
+from usl_lib.shared.wps_data import ML_REQUIRED_VARS_REPO, ScalingType, Unit
```

Comment:
the style guide sez:
"Use import statements for packages and modules only, not for individual types, classes, or functions."

https://google.github.io/styleguide/pyguide.html#22-imports

as does the zen of Python:

```
$ python3
Python 3.11.8 (main, Feb  7 2024, 21:52:08) [GCC 13.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import this
The Zen of Python, by Tim Peters
...
Namespaces are one honking great idea -- let's do more of those!
```